bin/ci_check

```diff
@@ -298,7 +298,7 @@ if (uvm):
                    if (debug):
```

Comment:
Is this PR intentionally to your fork instead of the openhw repo?

Reply:
Yes.  Didn't want to risk anything going horribly wrong.